tools/postman/scripts/transform-for-api.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+#########################################################
+# Prepare collection for Postman API
+# Environment variables:
+#   COLLECTION_FILE_NAME - name of the postman collection file
+#   COLLECTION_TRANSFORMED_FILE_NAME - name of the transformed collection file
+#   OPENAPI_FOLDER - folder where openapi file is saved
+#   TMP_FOLDER - folder for temporary files during transformations
+#   USE_ENVIRONMENT_AUTH - Bool for if auth variables are stored at the environment or collection level
+#   VERSIONS_FILE - name for the openapi versions file
+#########################################################
+
+COLLECTION_FILE_NAME=${COLLECTION_FILE_NAME:-"collection.json"}
+COLLECTION_TRANSFORMED_FILE_NAME=${COLLECTION_TRANSFORMED_FILE_NAME:-"collection-transformed.json"}
+OPENAPI_FOLDER=${OPENAPI_FOLDER:-"../openapi"}
+TMP_FOLDER=${TMP_FOLDER:-"../tmp"}
+USE_ENVIRONMENT_AUTH=${USE_ENVIRONMENT_AUTH:-true}
+VERSIONS_FILE=${VERSIONS_FILE:-"versions.json"}
+
+current_api_revision=$(jq -r '.versions."2.0" | .[-1]' < "${OPENAPI_FOLDER}/${VERSIONS_FILE}")
+
+pushd "${TMP_FOLDER}"
+
+echo "Wrapping Collection in \"collection\" tag"
+jq '{"collection": .}' "$COLLECTION_FILE_NAME" > intermediate1.json
+
+echo "Disabling query params by default"
+jq '(.. | select(.request? != null).request.url.query.[].disabled) = true ' intermediate1.json > intermediate2.json
+
+# This is to be removed because it is autogenerated when a new collection is created
+echo "Removing _postman_id"
+jq 'del(.collection.info._postman_id)' intermediate2.json > intermediate1.json
+
+echo "Updating name with version"
+jq '.collection.info.name = "MongoDB Atlas Administration API '"${current_api_revision}"'"' intermediate1.json >  intermediate2.json
+
+echo "DEV: Updating baseurl to cloud-dev.mongodb.com"
+jq '.collection.variable.[0].value = "cloud-dev.mongodb.com"' intermediate2.json > intermediate1.json
+
+if [ "$USE_ENVIRONMENT_AUTH" = "false" ]; then
+  echo "Adding auth variables"
+  jq '.collection.variable += [{"key": "digestAuthUsername", "value": "<string>"},
+  {"key": "digestAuthPassword", "value": "<string>"},
+  {"key": "realm", "value": "<string>"}]' intermediate1.json > "$COLLECTION_TRANSFORMED_FILE_NAME"
+else
+  cp intermediate1.json "$COLLECTION_TRANSFORMED_FILE_NAME"
```

Comment:
Consider naming that adds `collection` in the name. I do not mind any format. It can be even
```suggestion
  cp advancedCollection1.json "$COLLECTION_TRANSFORMED_FILE_NAME"
```

Reply:
renamed to intermediateCollection